CONTRIBUTING.md

```diff
@@ -1,5 +1,16 @@
 # Contributing notes
 
+We're looking forward to the new contributors.
+
+Our experts must verify all contributions from contributors. 
+
+We'd love it if you could contribute:
+ * New dashboard or improve existing ones.
+ * New widget or improve it. 
```

Comment:
I suggest to remove this line. -> "New widget or improve it. "
It would be a pain in a neck if a new widget is added and we would have to maintain it for all new Grafana releases. The same is about improvements. Are we going to deal with widget changes  when a new version of an original widget is released?